### DISCUSSION TITLE
How to fully delete a goddamn repository

### DISCUSSION THREAD

**Discussion by LordBloodfart** [community/community#22494] [New to GitHub]
<p>i clicked everything, theres one button that says delete this repository and i have done that multiple times but the repository is still there. If i google for solutions a " danger zone" is mentioned but that also is not where it is supposed to be. Now, i can not even navigate to that button anymore, its gone</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>this one ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/0/803109cf186c5c0674ab387fb81c84675bae52c5.png" data-download-href="/uploads/short-url/ii2bbaCSErbWE0s5nNKctQZVHSZ.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071445-649b8f38-f599-481f-8e37-4e86ff3a4c24.png" alt="image" data-base62-sha1="ii2bbaCSErbWE0s5nNKctQZVHSZ" width="505" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/803109cf186c5c0674ab387fb81c84675bae52c5_2_505x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/0/803109cf186c5c0674ab387fb81c84675bae52c5_2_757x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/0/803109cf186c5c0674ab387fb81c84675bae52c5_2_1010x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/803109cf186c5c0674ab387fb81c84675bae52c5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1030×1019 43 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>go to Repository settings, then at the bottom, click <code>Delete this repository</code></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/a/1a0607c9632c21459ad193020a408a207d799ee3.png" data-download-href="/uploads/short-url/3IdkJcxwCVp1A26vNL1q4YbAXET.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071447-628ab2d1-9fb8-47e4-a169-15240bbd9862.png" alt="image" data-base62-sha1="3IdkJcxwCVp1A26vNL1q4YbAXET" width="690" height="435" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/a/1a0607c9632c21459ad193020a408a207d799ee3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">700×442 20.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>you will be prompted</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181071449-d5c87f3f-06de-4b69-a0a5-b3f5e914c6fc.png" alt="image" data-base62-sha1="uHCdYoB2sSf1vzVmepcPdq10Gzr" width="505" height="413"></p>
<p>make sure you type that correctly then click the button below and mostly you will be prompted by a password input</p>

**Comment by LordBloodfart** [comment]
<p>thank you, i was super smart again.</p>
